### ISSUE TITLE
Unable to launch in Linux (Raspian)

### ISSUE THREAD

**Issue by marioandluigi878** [EldinZenderink/LittleWeeb#8]
So as the title says when I attempt to run the exe file with mono I get this 
`pi@raspberrypi:~/Desktop/LittleWeeb $ mono LittleWeebServer.exe
Welcome to littleweeb's server!
=================================
You can load the interface on your device which should be connected to the same network as where this server is running, using the following address:
When you use this application, you agree to the Terms of Use, which you can read on the About page. 
 This application is still in development, so many issues can occur! 
 Please report them here: https://github.com/EldinZenderink/LittleWeeb/issues
http://192.168.1.141:6010
Thanks for using LittleWeeb, if issues occur, please notify the developer here:
https://github.com/EldinZenderink/LittleWeeb/issues
DEBUG-MAIN: Starting to debug!
System.TypeLoadException: Could not load type 'WebSocketSharp.Server.WebSocketServer' from assembly 'websocket-sharp, Version=1.0.2.59611, Culture=neutral, PublicKeyToken=5660b08a1845a91e'.
  at LittleWeebLibrary.LittleWeebInit..ctor (System.Boolean local) [0x00024] in <a1a7ca4c0a61420a85c1574d710f7ee8>:0 
  at LittleWeebServer.Program.Main (System.String[] args) [0x000ea] in <feeba31675874d9b88140c2909c726e7>:0 
Could not start LittleWeeb!
Press a key to exit!
`
Nothing in the log file (which sucks)

**Comment by EldinZenderink** [comment]
Welp i didnt test it on the rpi... i somewhat thaught that the library that i used to communicate to the interface (websocket-sharp) ran on ARM... i suppose it doesnt :/ (specuating here atm) I will look into it when I am able to work on it (soon)

**Comment by EldinZenderink** [comment]
Just to be sure, in the location of LittleWeebServer.exe, does the file websocket-sharp.dll exists? Because, looking at the developer page, it says it supports mono, which should include every system that supports mono to my believe.

**Comment by marioandluigi878** [comment]
yes it does

**Comment by EldinZenderink** [comment]
Alright, possibly fixed the issue, had to do with the way i build my programs (both in one solution) and how libraries are shared... probably.

https://github.com/EldinZenderink/LittleWeeb/releases/tag/v0.3.1

You can redownload it here, if it still doesn't work, please notify me. Then the issue is probably more complicated than I hoped ;(.

**Comment by marioandluigi878** [comment]
You sure thats the new one cause it seems like the same version i downloaded

**Comment by marioandluigi878** [comment]
You sure thats the new one cause it seems like the same version i downloaded

**Comment by EldinZenderink** [comment]
Yes, i just updated the files since code wise nothing has changed hence i dont feel the need to use a different version number.

**Comment by marioandluigi878** [comment]
Same problem
pi@raspberrypi:~/Downloads/test $ mono LittleWeebServer.exe
Welcome to littleweeb's server!
=================================
You can load the interface on your device which should be connected to the same network as where this server is running, using the following address:
When you use this application, you agree to the Terms of Use, which you can read on the About page. 
 This application is still in development, so many issues can occur! 
 Please report them here: https://github.com/EldinZenderink/LittleWeeb/issues
http://192.168.1.141:6010
Thanks for using LittleWeeb, if issues occur, please notify the developer here:
https://github.com/EldinZenderink/LittleWeeb/issues
DEBUG-MAIN: Starting to debug!
System.TypeLoadException: Could not load type 'WebSocketSharp.Server.WebSocketServer' from assembly 'websocket-sharp, Version=1.0.2.59611, Culture=neutral, PublicKeyToken=5660b08a1845a91e'.
  at LittleWeebLibrary.LittleWeebInit..ctor (System.Boolean local) [0x00024] in <e496ac8207ec4ef6826a0db3804e724b>:0 
  at LittleWeebServer.Program.Main (System.String[] args) [0x000ea] in <91cf624113ab4b5e87517f0d504fd300>:0 
Could not start LittleWeeb!
Press a key to exit!

**Comment by EldinZenderink** [comment]
Hi, version 0.4.0 has been released for most platforms: https://littleweeb.github.io/ or https://github.com/littleweeb/Desktop, should fix the issues you were having entirely ;)
